tests/playwright/src/ai-lab-extension.spec.ts

```diff
@@ -145,6 +146,51 @@ test.describe.serial(`AI Lab extension installation and verification`, { tag: '@
     });
   });
 
+  ['ggerganov/whisper.cpp'].forEach(modelName => {
+    test.describe.serial(`Model service creation and deletion`, () => {
+      let catalogPage: AILabCatalogPage;
+      let modelServiceDetailsPage: AILabServiceDetailsPage;
+
+      test.skip(isLinux, `Skipping model service creation on Linux`);
+      test.beforeAll(`Open AI Lab Catalog`, async ({ runner, page, navigationBar }) => {
+        [page, webview] = await handleWebview(runner, page, navigationBar);
+        aiLabPage = new AILabPage(page, webview);
+        await aiLabPage.navigationBar.waitForLoad();
+
+        catalogPage = await aiLabPage.navigationBar.openCatalog();
+        await catalogPage.waitForLoad();
+      });
+
+      test(`Download ${modelName} model if not available`, async () => {
+        test.setTimeout(310_000);
+        if (!(await catalogPage.isModelDownloaded(modelName))) {
+          await catalogPage.downloadModel(modelName);
+        }
+        await playExpect
+          // eslint-disable-next-line sonarjs/no-nested-functions
+          .poll(async () => await waitForCatalogModel(modelName), { timeout: 300_000, intervals: [5_000] })
+          .toBeTruthy();
+      });
+
+      test(`Create model service for ${modelName}`, async () => {
```

Comment:
please add information that we are creating a service from catalog directly. `Create model service from catalog for modelName`.

Reply:
done